awsagentprovider/src/test/java/software/amazon/opentelemetry/javaagent/providers/AwsMetricAttributeGeneratorTest.java

```diff
@@ -1029,6 +1030,84 @@ private void validateHttpStatusForNonLocalRootWithThrowableForClient(
     }
   }
 
+  @Test
+  public void testDBUserAttribute() {
+    mockAttribute(DB_SYSTEM, "db_system");
+    mockAttribute(DB_OPERATION, "db_operation");
+    mockAttribute(DB_STATEMENT, "db_statement");
```

Comment:
Not blocking: Only need to set one of these.
```suggestion
    mockAttribute(DB_STATEMENT, "db_statement");
```